engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -916,42 +916,35 @@ namespace dmGameSystem
             const uint32_t* page_indices = texture_set_ddf->m_PageIndices.m_Data;
             const dmGameSystemDDF::SpriteGeometry* geometries = texture_set_ddf->m_Geometries.m_Data;
 
+            uint32_t* anim_index = data->m_Resources[i]->m_AnimationIds.Get(anim_id);
```

Comment:
The animation itself is named the same in the textures.
No need to do it separately (as was done previously)